deploy/injector-mutating-webhook.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
what about people who install via the helm chart

Reply:
via helm chart is v1beta1.
https://github.com/hashicorp/vault-helm/blob/master/templates/injector-mutating-webhook.yaml
It can be enhanced to use both.